fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -209,6 +210,9 @@ private const val BottomSheetOpenFraction = 0.5f
  * @param slideOver if true, then sheetContent would be drawn in full length & it just get slided
  * in the visible region. If false then, the sheetContainer placed at the bottom & its height could be at peekHeight, fullheight or hidden when dragged by Handle or swipe down.
  * @param enableSwipeDismiss if false, bottomSheet will not be dismissed after swipe down gesture. Default value is false.
+ * @param stickyBehavior if true, bottomSheet will stick to the Expanded and Shown State when dragged beyond the stickyThresholdUpward or stickyThresholdDownward. Default value is false. When it's false the default Up and Down Drag Threshold value is 56dp
```

Comment:
Rephrase to - "If true, Bottomsheet will change the state to Expanded or Shown state only when dragged beyond the stickyThresholdUpward or stickyThresholdDownward. When false the default value for Upward and downward Thresholds are 56.dp"

Reply:
Removing this param